rules/DowngradePhp56/Rector/Use_/DowngradeUseFunctionRector.php

```diff
@@ -76,6 +87,24 @@ private function refactorUse(Use_ $use): void
         }
     }
 
+    private function isAlreadyFullyQualified(ConstFetch|FuncCall $node): bool
+    {
+        $file = $this->currentFileProvider->getFile();
+        if (! $file instanceof File) {
+            return false;
+        }
+
+        $oldTokens = $file->getOldTokens();
```

Comment:
`$file` property can be accessed instead:

```suggestion
        $oldTokens = $this->file->getOldTokens();
```